.azure/pr-check.yaml

```diff
@@ -23,5 +30,39 @@ steps:
       python .azure/scripts/validate_samples.py
     displayName: "Metadata Validation"
 
+  # Give execute permissions to trivy-scan.sh
+  - script: |
+      echo "Giving execute permission to trivy-scan.sh"
+      chmod +x .azure/scripts/trivy-scan.sh
+    displayName: "Set execute permissions"
+
+  - template: install/install-trivy.yaml@common-templates
+  - script: |
```

Comment:
Shall we remove the comments where the code is obvious? The code will look cleaner that way.